samples/android-integration/launching-activities/src/main/java/com/badoo/ribs/samples/android_integration/launching_activities/rib/builder/LaunchingActivitiesChildBuilders.kt

```diff
@@ -0,0 +1,14 @@
+package com.badoo.ribs.samples.android_integration.launching_activities.rib.builder
```

Comment:
Package structure seems merged.

It should be:
`.rib` (a package to hold all ribs defined in this module, each in its own subpackage)
`.rib.parent` (A single rib)
`.rib.child` (Another rib)
and no `.builder` subpackage if it only contains a single file.

Reply:
It's not the package structure that is merged. I used the same node class for both parent and child.
I have changed it. Now it is a parent that hosts two children. The new package structure closely resembles the routing example.